src/locales/en/finance/currency.ts

```diff
@@ -88,8 +88,8 @@ export default {
     symbol: 'P',
   },
   'Belarussian Ruble': {
```

Comment:
```suggestion
  'Belarusian Ruble': {
```
Somewhat unrelated, but shouldn't this be "Belarusian" and not "Belarussian"?

Reply:
Jup, you want me to add a commit to fix that or add it to the current commit?

Reply:
Please provide an additional PR for that fix so its easier to understand in the commit history (and thus in release notes).

Reply:
Done https://github.com/faker-js/faker/pull/1395